### PR TITLE
0.9.3 typo fixes

### LINKED ISSUE #4034: 092 + 093 typos
w

### PR DESCRIPTION
**Pull request by MisterSimple**
#4034

### CODE REVIEW

**Comment by multimokia** [comment]
Once again, keeping this open until closer to release in case any more typos pop up last minute.

File: Monika After Story/game/script-topics.rpy [hunk 1]
```diff
@@ -10093,12 +10093,12 @@ init 5 python:
...
     m 1eua "Hey [player], have you ever read {i}The Metamorphoses{/i}?"
-    m 3eua "It’s a collection of poems by an ancient Roman poet named Ovid."
-    m 1eud "There’s one poem that really struck me, called {i}The Story of Pygmalion and the Statue.{/i}"
+    m 3eua "It's a collection of poems by an ancient Roman poet named Ovid."
+    m 1eud "There's one poem that really struck me, called {i}The Story of Pygmalion and the Statue.{/i}"
     m 1eua "It tells the story of a sculptor, Pygmalion, who fell in love with an ivory statue that he had sculpted."
     m 3eua "He loved the statue so much, that during one of the festivals held for Aphrodite, he made an offering to the goddess and wished for her to grant life to the ivory statue."
     m 1eub "When he returned to his home, he kissed the statue and to his surprise, the lips of the statue felt warm."
-    m 1eua "He kissed the statue once more, and the ivory of statue lost its hardness, becoming soft like skin."
+    m 1eua "He kissed the statue once more, and the ivory lost its hardness, becoming soft like skin."
```
Comment (Rai99) [inline Monika After Story/game/script-topics.rpy]: Could we change this to "He kissed the statue once more, and the ivory _of the statue_ lost its hardness, becoming soft like skin."?

Comment (MisterSimple) [reply Monika After Story/game/script-topics.rpy]: I considered that option as well, but I think that adding "of the statue" doesn't really contribute anything meaningful to the line, as we already know we're talking about a statue.

Comment (multimokia) [reply Monika After Story/game/script-topics.rpy]: In terms of flow,I think it works better. Would rather that change personally

Comment (MisterSimple) [reply Monika After Story/game/script-topics.rpy]: Discussed this @multimokia and @jmwall24, they approve of my change.

**Pull request merged**